docs/guides/scripts.md

```diff
@@ -55,6 +55,15 @@ $ uv run example.py hello world!
 hello world!
 ```
 
+Additionally, your script can be read directly from stdin:
+
+```console
+$ uv run - <<EOF
+print("hello world!")
+EOF
+hello world!
+```
```

Comment:
Minor problem, the copy paste button only grabs `uv run - <<EOF` because the rest is treated as output. Maybe we should do

```suggestion
```bash
uv run - <<EOF
print("hello world!")
EOF
```
```

Reply:
(GitHub removed the trailing ``` in the suggestion, will need to be added)

Reply:
oh gotcha ok. and should we leave the actual output off the doc?

Reply:
I think that's the best option here.

Reply:
all g, done